1-exercises/G-array-methods/exercise2.js

```diff
@@ -7,13 +7,15 @@
 let mentors = ["Daniel", "Irina", "Rares"];
 let students = ["Rukmini", "Abdul", "Austine", "Swathi"];
 
-let everyone; // complete this statement
+let  firstFive = everyone.slice(0.5); // complete this statement
+let  lastFive = everyone.slice(-5);
```

Comment:
The everyone array doesn't exist in this file, you need to use concat to create it, that was the goal of the excercise I'm afraid